app/controllers/api/V1/todo_lists_controller.rb

```diff
@@ -53,44 +41,33 @@ def edit
 
       # POST /todolists or /todolists.json
       def create
-        @todo_list = TodoList.new(todo_list_params)#
-        #@todo_list = Todolist.new(todo_list_params)
+        @todo_list = TodoList.new(todo_list_params)
         if @todo_list.save
-            #flash[:topnotice] = 'New todo list has been created successfully.'                
-              #format.html { redirect_to todo_lists_path }
-              render json: {message: "New Todolist is created successfully. ", status: :ok }
+              render json: {message: "New Todolist is created successfully. "}, status: :ok 
         else
-              format.json { render json: @todo_list.errors.to_json, status: :error}
+          render json: { message: @todo_list.errors, status: :error}
         end
 
       end
 
       # PATCH/PUT /todolists/1 or /todolists/1.json
-      def update
-        respond_to do |format|
-          
+      def update          
           if @todo_list.update(todo_list_params)
-            render json: {message: "Updated successfully.", status: :ok }
-            # format.html { redirect_to todolist_url(@todo_list), notice:  }
-            # format.json { render :show, status: :ok, location: @todo_list }
+            render json: {message: "Updated successfully."}, status: :ok 
           else
-            format.html { render :edit, status: :unprocessable_entity }
-            format.json { render json: @todo_list.errors, status: :unprocessable_entity }
+            render json: { message: @todo_list.errors}, status: :unprocessable_entity 
           end
-        end
       end
+      
 
       # DELETE /todolists/1 or /todolists/1.json
       def destroy
-        p "deleting id = #{params[:id]}"
         @todo_list_todelete = TodoList.find(params[:id])
-        p @todo_list_todelete.title
         if @todo_list_todelete.destroy  
```

Comment:
should use destroy! instead of destroy
see the difference: [https://apidock.com/rails/ActiveRecord/Persistence/destroy%21](https://apidock.com/rails/ActiveRecord/Persistence/destroy%21)